project/plugins.sbt

```
@@ -2,6 +2,6 @@ resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefac
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.3.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.1.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.8.7")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.20.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.5.0")
+addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.1")
```

```suggestion
addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.1")
addSbtPlugin("com.timushev.sbt"  % "sbt-updates"        % "0.6.4")
```

Worth adding the updates plugin too